README.md

```diff
@@ -99,28 +99,32 @@ It should take less than 15 minutes to install (depends on your machine).
 Customize in `~/.laptop.local`
 ------------------------------
 
-Your `~/.laptop.local` is run at the end of the Laptop script.
+Your `~/.laptop.local` is run near the end of the Laptop script.
```

Comment:
It looks like the very last thing we do in the `mac` script  is run `~/.laptop.local`. What am I missing?